sql/moz-fx-data-shared-prod/backfills_staging_derived/baseline_clients_last_seen_v1_20211130/query.sql

```diff
@@ -0,0 +1,98 @@
+-- Generated via bigquery_etl.glean_usage
+{% if is_init() %}
+  SELECT
+    CAST(NULL AS INT64) AS days_seen_bits,
+    CAST(NULL AS INT64) AS days_active_bits,
+    CAST(NULL AS INT64) AS days_created_profile_bits,
+    CAST(NULL AS INT64) AS days_desktop_active_bits,
+  -- We make sure to delay * until the end so that as new columns are added
+  -- to the daily table we can add those columns in the same order to the end
+  -- of this schema, which may be necessary for the daily join query between
+  -- the two tables to validate.
+    *
+  FROM
+    `firefox_desktop_derived.baseline_clients_daily_v1`
```

Comment:
The not-fully-qualified table references and UDF references in this ETL is causing the `verify-format-sql` CI check to fail with
```
Needs reformatting: bqetl format sql/moz-fx-data-shared-prod/backfills_staging_derived/baseline_clients_last_seen_v1_20211130/query.sql
1 file would be reformatted, 1757 files would be left unchanged.
```
To fix, run the `bqetl format` command on this file`.